### ISSUE TITLE
Double printing first operand

### ISSUE THREAD

**Issue by nathanhjay** [dyninst/dyninst#193]
@jdetter we seem to be printing a single operand twice instead of printing the second operand properly.
```
Bytes: 0f 2c 44 81 4e
Dyninst output: cvttps2pi 0x4e(%rcx,%rax,4),0x4e(%rcx,%rax,4): 
Assembler error: too many memory references for 'cvttps2pi'
Expected output: cvttps2pi 0x4e(%rcx, %rax, 4), %mm0: ERROR:  bad register name 
```

**Comment by jdetter** [comment]
These issues are blocked by #210 for now, There will probably be new issues after the syntax changes are merged in but hopefully the syntax changes will fix some of the issues. I will reevaluate the issues after I have the x86 syntax stuff working again on the AT&T branch.

Also I wouldn't pull from the AT&T branch right now because the x86 stuff has all been commented out so you will get empty strings for all instructions.

**Comment by jdetter** [comment]
This took me forever to fix because the operands were completely wrong and I had to look them up in the manual and I put in the right ones but we don't actually support one of those addressing modes and instead we use an equivalent addressing mode that is similar.

Basically, this is fixed. I'll look for similar issues.

All of my changes are going on a separate branch that we will probably merge into master.
